### PR TITLE
Main

### PR DESCRIPTION
**Pull request by elliotmitchum**
Refactors `Main` class.

Criteria:

- Evaluate against the class diagram
- Evaluate against the code guidelines
- Project builds and runs

### CODE REVIEW

**Comment by elliotmitchum** [comment]
@amrmosit This is ready for review

**Comment by amrmosit** [comment]
@haynescrossley branch is ready for you to review.

File: src/library/Main.java [hunk 1]
```diff
@@ -205,72 +212,70 @@ private static void ADD_ITEM() {
...
-            String LaSt_NaMe = input("Enter last name: ");
-            String EmAiL_AdDrEsS = input("Enter email address: ");
-            long PhOnE_NuMbEr = Long.valueOf(input("Enter phone number: ")).intValue();
-            Patron PaTrOn = LIBRARY.addPatron(FiRsT_NaMe, LaSt_NaMe, EmAiL_AdDrEsS, PhOnE_NuMbEr);
-            output("\n" + PaTrOn + "\n");
-            
-        } catch (NumberFormatException e) {
-             output("\nInvalid phone number\n");
+            String firstName = input("Enter first name: ");
+            String lastNae = input("Enter last name: ");
```
Comment (haynescrossley) [inline src/library/Main.java]: #PR43 - D1 : typo in variable name. Please rename to 'lastName'

File: src/library/Main.java [hunk 2]
```diff
@@ -15,184 +16,190 @@
...
+        new FixItemUI(new fIX_iTeM_cONTROL()).RuN();
     }
 
 
-    private static void INCREMENT_DATE() {
+    private static void incrementDate() {
         try {
+            Date currentDate = CALENDAR.getDate();
+            // @todo Fix method invocation as parameter.
             int days = Integer.valueOf(input("Enter number of days: ")).intValue();
```
Comment (haynescrossley) [inline src/library/Main.java]: #PR43 - D2 : I think this breaches CSG 10 about the argument list of methods must not contain other method calls?

Is .intValue() even necessary on the end here?

Maybe something like this?
String input = input("Enter number of days: ");
int days = Integer.parseInt(input);

File: src/library/Main.java (hunk 1 above)
Comment (elliotmitchum) [reply src/library/Main.java]: https://github.com/elliotmitchum/ITC205/pull/43/commits/d9135d6585e083ab375330a2c76a02385a4740b8

File: src/library/Main.java (hunk 2 above)
Comment (elliotmitchum) [reply src/library/Main.java]: https://github.com/elliotmitchum/ITC205/pull/43/commits/2d861b103c9693ee18c9024a472f3d9116c49fc3

**Review by haynescrossley — APPROVED** [review]
Only 2 defects found which have been resolved.

Approved to merge @amrmosit

**Comment by amrmosit** [comment]
Master checked after merge and is still operational

**Pull request merged**